README.md

```diff
@@ -21,7 +23,16 @@ Ensure the following tools are installed and present in your `$PATH`:
 * [.NET](https://dotnet.microsoft.com/download)
 
 
-### Creating and Initializing the Repository
+### Test the boilerplate XYZ provider before making changes
```

Comment:
Is this for contributors to this repo or consumers of this template?

Reply:
The intent is to improve the repo for developers who will consume this template.

Reply:
reduced this section title https://github.com/pulumi/pulumi-provider-boilerplate/pull/80/commits/99977e08fe797a63a23e852bc566f475aea4cbf7